tests/estimators/certification/test_smooth_vit.py

```diff
@@ -0,0 +1,616 @@
+# MIT License
```

Comment:
Please update file name to `test_vision_transformers.py`

Reply:
renamed